ksqldb-parser/src/main/java/io/confluent/ksql/parser/SyntaxErrorValidator.java

```diff
@@ -43,6 +47,9 @@ public class SyntaxErrorValidator extends BaseErrorListener {
       "extraneous input.*expecting.*");
   private static final Pattern MISMATCHED_INPUT_PATTERN = Pattern.compile(
       "mismatched input.*expecting.*");
+  private final List<String> commands = Arrays.asList("SELECT", "CREATE", "INSERT", "DESCRIBE",
```

Comment:
Would it be possible to use the `GrammarTokenExporter` instead? It returns more tokens than just the commands, but it would prevent people from having to update extra lists when they add new syntax to ksqlDB.